modules/social_features/social_group/src/SocialGroupHelperService.php

```diff
@@ -60,4 +53,33 @@ public static function getGroupFromEntity($entity) {
     return $gid;
   }
 
+  /**
+   * Returns the default visibility.
+   *
+   * @param string $type
+   *   The Group Type.
+   *
+   * @return string
+   *   The default visibility.
+   */
+  public static function getDefaultGroupVisibility($type) {
```

Comment:
Not entirely sure, but don't we want to use `&drupal_static(__FUNCTION__);` here as well?

Reply:
The alter hook only has to be executed once per request anyway.